src/pages/p2p/components/_hero.tsx

```diff
@@ -156,16 +156,40 @@ const TryButton = styled(LinkButton)`
     }
 `
 
-const HeroComponent = ({ title, content, background_data, img_data }) => {
+const query = graphql`
+    query {
+        p2p_hero_background: file(relativePath: { eq: "p2p/p2p_hero_background.png" }) {
+            ...fadeIn
+        }
+        p2p_hero_background_mobile: file(
+            relativePath: { eq: "p2p/p2p_hero_background_mobile.png" }
+        ) {
+            ...fadeIn
+        }
+        p2p_hero_img: file(relativePath: { eq: "p2p/p2p_hero_img.png" }) {
+            ...fadeIn
+        }
+    }
+`
+
+const Hero = () => {
+    const data = useStaticQuery(query)
+    const [is_mobile] = useBrowserResize()
```

Comment:
You need to pass in `size.tabletL` to be consistent with the 992 breakpoint you are refactoring it from.

Reply:
we don't have this in usebrowser hook

Reply:
It accepts a size argument. You can pass in like `useBrowserResize(size.tabletL)`.